i18n.ts

```diff
@@ -0,0 +1,12 @@
+import { notFound } from "next/navigation";
+import { getRequestConfig } from "next-intl/server";
+import { locales } from "./i18nconfig";
+
+export default getRequestConfig(async ({ locale }) => {
+  // Validate that the incoming `locale` parameter is valid
+  if (!locales.includes(locale as any)) notFound();
```

Comment:
In `Next.js` is there an option to include a message in `notFound()`? If so that would be useful, something like:
```suggestion
  if (!locales.includes(locale as any)) notFound(`Unsupported locale '${ locale }`);
```